RegisteredAccountsScreen

Jenna-Zing opened the pull request:
has constraints

Commit:
prototype

has constraints

Commit:
Merge branch 'master' into RegisteredAccounts

Commit:
added firebase

set up firebase

Joanne-Chang commented:
Hey, thanks for setting up the Firebase. 😊 

I did want to make a note that the project seems to be referencing the `GoogleService-Info.plist` from the local computer and not the one in the actual project folder.

<img width="892" alt="Screen Shot 2020-05-24 at 8 22 09 AM" src="https://user-images.githubusercontent.com/23194620/82758590-4073a080-9d9c-11ea-961e-3bdad296a56f.png">

Could you please change this? You can do so like this:

![Screen Shot 2020-05-24 at 9 02 44 AM](https://user-images.githubusercontent.com/23194620/82758894-25099500-9d9e-11ea-834d-97812e43b1da.png)

Joanne-Chang commented on PasswordManager/PasswordManager.xcodeproj/project.pbxproj:
```diff
@@ -7,15 +7,23 @@
 	objects = {
```
Since this file is automatically generated by Xcode, should we probably not be committing this from individual branches? We might need what's in this for Firebase, though. (And then maybe manually add in the lines needed for the other screens to it.) Same with the workspace file, but I'm not entirely sure. What's everyone's thoughts on this?

evelynsj replied:
I think that maybe we can take this out for now and put it back in to the project if we need it

Joanne-Chang commented on PasswordManager/PasswordManager.xcodeproj/xcuserdata/Kurono.xcuserdatad/xcschemes/xcschememanagement.plist:
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```
Just wondering, what is this file for?

Joanne-Chang reviewed (commented):
Looks good so far, I have a few questions though.

Commit:
fixed firebase reference

GoogleService-info.plist is now in the project and reference should be fixed in XCode.

VivZing committed:
up to date with master

Jenna-Zing commented:
- initial design for screen
- implemented initial firebase setup and installed cocoapods
- further refined Firebase Database model representations
